### PR TITLE
PBM-577 Doc S3 storage class support

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/configuration-options.rst
	modified:   source/installation.rst
	modified:   source/storage-configuration.rst

### CODE REVIEW

File: source/storage-configuration.rst [hunk 1]
```diff
@@ -33,11 +33,26 @@ As of v1.3.2, |PBM| supports :term:`server-side encryption <Server-side encrypti
...
+As of version 1.7.0, |PBM| supports `Amazon S3 storage classes <https://aws.amazon.com/s3/storage-classes/>`_. Knowing your data access patterns, you can set the S3 storage class in |PBM| configuration. When |PBM| uploads data to S3, the data is distributed to the corresponding storage class. The support of S3 bucket storage types allows you to effectively manage S3 storage space and costs.
+
+To set the storage class, specify the ``storage.s3.storageClass`` option in |PBM| configuration file
+
+.. code-block:: yaml
+
+   storage:
+     type: s3
+     s3:
+       storageClass: INTELLIGENT_TIERING
```
Comment (spron-in) [inline source/storage-configuration.rst:45]: If AWS adds new storage class - will we need to change our code or we just pass through this option? @defbin

File: source/configuration-options.rst [hunk 2]
```diff
@@ -136,12 +140,12 @@ It can be useful when using an S3 provider that supports a smaller number of chu
...
 
 storage.s3.storageClass
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 :type: string
 :required: NO
 
-The storage class assigned to objects stored in the S3 bucket. If not provided, the ``STANDARD`` storage class will be used.
+The `storage class <https://aws.amazon.com/s3/storage-classes/>`_ assigned to objects stored in the Amazon S3 bucket. If not provided, the ``STANDARD`` storage class will be used. This option is available in |PBM| as of v1.7.0.
```
Comment (spron-in) [inline source/configuration-options.rst:148]: Is it only AWS?

Comment (defbin) [reply source/configuration-options.rst:148]: minio supports 2 storage classes: STANDARD and REDUCED_REDUNDANCY

https://github.com/minio/minio/blob/master/docs/erasure/storage-class/README.md

File: source/storage-configuration.rst (hunk 1 above)
Comment (defbin) [reply source/storage-configuration.rst:45]: no. via passthrough the value

**Review by defbin — APPROVED** [review]

**Pull request merged**